### ISSUE TITLE
Ghast Trap ghastlings instantly die

### ISSUE THREAD

**Issue by TherminatorX** [Electroblob77/TF-Spell-Pack#11]
The Ghastlings spawned by the Ghast Trap spell die immediately to 'magic', presumably the lighting strike accompanying them.

Version 1.0, though nothing about this is in the changelogs for the updated version

**Comment by Electroblob77** [comment]
Hmm, I thought I'd made them immune to that. I'll have to take a look, thanks.

**Comment by Electroblob77** [comment]
I can't reproduce this issue on my end. The ghastlings are summoned as normal and fly about at full health.

**Comment by TherminatorX** [comment]
Reproducible for sure in 1.0.1, will update in a short while and see if it's somehow fixed
![image](https://user-images.githubusercontent.com/7256249/88605168-c9c67e00-d03e-11ea-8e77-fda119af6ad4.png)

**Comment by TherminatorX** [comment]
Updated to 1.0.2, new world, same issue
![image](https://user-images.githubusercontent.com/7256249/88606241-cb457580-d041-11ea-8b79-73128e4f46e9.png)

**Comment by TherminatorX** [comment]
Forge version is 14.23.5.2854 on those, by the way

**Comment by TherminatorX** [comment]
Changed to a 'test set' modlist of only QoL and Wizardry related mods, Ghastlings are now flying about as intended 🤔 
Will check other mods to see what the problem might be.

**Comment by TherminatorX** [comment]
Okay never mind, trying to remove mods is crashing my game on startup and freezing the launcher to make the logs unreadable

**Comment by Volence** [comment]
Yeah getting the same issue currently

**Comment by TherminatorX** [comment]
This issue appears to be something to do with Inventory Pets?

**Comment by Volence** [comment]
Weird, looks like the lightning that's summoning them is killing them. Very rarely will one survive with really low health.
![image](https://user-images.githubusercontent.com/32358820/88697681-faf08e00-d0d2-11ea-808c-1f15338b189f.png)

**Comment by Electroblob77** [comment]
Right, sorry for the long delay but that information is helpful, thanks. TF spell pack sets the ghastlings to have a second or so of invulnerability on spawn to protect them from the lightning, using the vanilla system to do so. However, I have seen problems before caused by potion core messing with this system, so did you by any chance have potion core installed? If so, others have been able to solve previous issues by disabling a config option for potion core, I think it's called `separateInvulnerabilityTimers` or something similar.

**Comment by TherminatorX** [comment]
MC Eternal, the pack I first encountered this on, does not have Potion Core installed. The issue came up when installing Inventory Pets.
